web/src/components/overlay/SaveExportOverlay.tsx

```diff
@@ -17,7 +17,7 @@ export default function SaveExportOverlay({
   return (
     <div className={className}>
       <div
-        className={`flex justify-center px-2 gap-2 items-center pointer-events-auto rounded-lg *:text-white ${
```

Comment:
we only have two children here so might be better to just apply text white to the other button?